tests/cpp/test_gpu3.cpp

```diff
@@ -7385,9 +7385,9 @@ TEST_F(NVFuserTest, AllInputDtypes) {
     DataType ptr_type =
         PointerType{std::make_shared<DataType>(DataType::Float)};
     auto ptr = IrBuilder::create<Val>(ptr_type);
-    DataType array_type =
-        ArrayType{std::make_shared<DataType>(DataType::Float), 2};
-    auto array = IrBuilder::create<Val>(array_type);
+    // DataType array_type =
```

Comment:
@zasdfgbnm this was something I couldn't figure out how to get to work with KernelArgumentHolder, could you please advise what the right push incantation is for it?